### PR TITLE
Dynamixel v2

### PR DESCRIPTION
**Pull request by costashatz**
Fix for https://github.com/resibots/libdynamixel/pull/6 and https://github.com/resibots/hexapod_common/pull/5

### CODE REVIEW

File: hexapod_bringup/config/pexod.yaml [hunk 1]
```diff
@@ -73,24 +73,43 @@ dynamixel_control_hw:
...
+    "14": 3.14159265359
+    "24": 4.71238898038
+    "5": 3.14159265359
+    "15": 3.14159265359
+    "25": 4.71238898038
+    "6": 3.53429173529
+    "16": 3.14159265359
+    "26": 4.71238898038
+
+    # "1": -256
```
Comment (dogoepp) [inline hexapod_bringup/config/pexod.yaml]: Maybe these lines can be removed ?

Comment (costashatz) [inline hexapod_bringup/config/pexod.yaml]: Yeap. And some other comments in the CMake (for finding hexapod_controller)...

File: hexapod_driver/src/hexapod.cpp [hunk 2]
```diff
@@ -10,17 +14,26 @@ Hexapod::Hexapod(ros::NodeHandle nh, std::string ns) : _nh(nh), _namespace(ns)
...
     // Private node handle
     ros::NodeHandle n_p("~");
     // Load Server Parameters
-    n_p.param("Odom", _odom_topic_name, std::string("/odom"));
-    n_p.param("OdomEnable", _odom_enable, true);
-    n_p.param("MoCapOdomTransformEnable", _mocap_odom_enable, false);
+    n_p.param("Odom", _odom_frame, std::string("/odom"));
+    n_p.param("BaseLink", _base_link_frame, std::string("/base_link"));
+    n_p.param("OdomEnable", _odom_enable, false);
+    n_p.param("MoCapOdomEnable", _mocap_odom_enable, true);
```
Comment (dogoepp) [inline hexapod_driver/src/hexapod.cpp]: Is CamelCase the recommended naming scheme for parameters ? I forgot.

Comment (costashatz) [inline hexapod_driver/src/hexapod.cpp]: Not really. Check [here](http://wiki.ros.org/Names) for the official conventions.. We can change it to whatever you feel like..

Comment (dogoepp) [inline hexapod_driver/src/hexapod.cpp]: What about following the usual ROS names for topics ?

Comment (costashatz) [inline hexapod_driver/src/hexapod.cpp]: Which are? Feel free to suggest names...

Comment (dogoepp) [inline hexapod_driver/src/hexapod.cpp]: You make me wonder if I'm mistaken. Isn't it usually rather like
- odom
- odom_enable
- base_link
  mo_cap_odom_enable

Comment (costashatz) [inline hexapod_driver/src/hexapod.cpp]: I do not remember...I will make them like this..

**Comment by costashatz** [comment]
@dogoepp It should be okay now...

**Comment by dogoepp** [comment]
**OK** for me.

**Comment by costashatz** [comment]
**Merging...**

**Pull request merged**